### ISSUE TITLE
Lab 02 Question 6

### ISSUE THREAD

**Issue by castower** [DS4PS/cpp-523-fall-2019#9]
Hello all,

I am currently working on solving question 6 of Lab 2: "The covariance of class size and test scores is -418, and the variance of class size is 99. Can you calculate the slope of class size in Model 4 with the formula cov(x,y)/var(x)? Why or why not?"

As I've been working through the labs, my understanding is that b1 is the slope and the formula cov(x,y)/var(x) is another way to find the slope. However, I'm a bit stumped as my calculation to find cov(x,y)/var(x) using the numbers above does not match the calculated value for b1 listed for Model 4 in the table.

Specifically:

```{r}
slope <- round ( -418/99, 2 )
slope
```
does not match the -2.67 listed in the table.

Is there something that I'm missing here? 

Thanks!
Courtney

**Comment by castower** [comment]
Looking at the model again, I'm suspecting that it's possibly because this is not accounting for all of the variance in the model (there's an additional unaccounted for variable in this calculation), but I'm not 100% sure that's following the proper line of reasoning...

**Comment by lecy** [comment]
Note that we have five models with four different estimates for the Classroom Size slope:

![image](https://user-images.githubusercontent.com/1209099/64078625-c80b0380-cc91-11e9-9efa-96f96549410f.png)

We know that one formula cannot generate all four slopes, so which slope does the formula create? Which model has a slope of -4.22? 

This question is a preview of next week's topic on control variables.

**Comment by lecy** [comment]
Your reasoning is not far off. 

"I'm suspecting that it's possibly because this [formula] is not accounting for all of the variance in the model (there's an additional unaccounted for variable in this calculation)."

More accurately, though, there is less variance to account for after we add control variables. They soak up some of the variance, so there is less left to explain. 

The impact on the slope of our policy variable (classroom size) and the standard error of the policy variable are nuanced, but we will unpack all of that soon.

**Comment by JasonSills** [comment]
Hi, I used the Lewis-Beck chapter to answer this question. Page 49 offers some insight under "Interpreting the Parameter Estimates".

**Comment by cjbecerr** [comment]
I was struggling a bit with this question as well. I think I have the answer now and will reference the reading, but to check my thinking. We need to use an adjusted formula for model 4 because the SES is included.

**Comment by lecy** [comment]
Yep, this is a question to make you pause and note that the slope formula from lab-01 ( `cov(x,y)/var(x)` ) produces the slope in Model 1.

~~~ PREVIEW OF NEXT WEEK

The slope from Model 1 represents the most direct and basic relationship between X and Y (class size and test scores). 

When using observational data we can measure the relationship between our policy and outcome (the slope associated with classroom size in this case). But we cannot interpret it as causal impact because class size might just be a proxy for something else. Suburbs have smaller classes, for example, but also wealthier parents, more spending per pupil, and better teachers - so which factor is driving the outcome we observe if they are all related? Is it really class size? 

Adding controls will alter the variance of class size and test scores in a variety of ways, and thus change the slope and the standard errors. If we use control variables properly we can "fix" the slope so it is a better estimate of the true impact of the policy. 

At this point, the observation that adding controls changes the slope of our policy variable is sufficient. The formula `cov(x,y)/var(x)` offers us a description of how X and Y are moving together - it is the regression line that appears on a scatterplot of X and Y. 

But to get to a causal estimate of the impact of X on Y we need to account for other factors, which we do by adding controls. As we include other explanations as control variables we will see that our slope describing program impact will often change (as it does in this case).
